src/employee/employee.service.ts

```diff
@@ -0,0 +1,51 @@
+import { IEmployeeRepository } from "./employee.repository";
+import { EmployeeType, EmployeeCreateType } from "./interfaces";
+import { IRepositories } from "../repositories";
+
+export interface IEmployeeService {
+  createEmployee: (employee: EmployeeCreateType) => Promise<EmployeeType>;
+  getAllEmployees: () => Promise<EmployeeType[]>;
+  getEmployeeById: (id: number) => Promise<EmployeeType | undefined>;
```

Comment:
Z semantyki możesz się pozbyć Employee (już wiesz że to jest serwis za to odpowiedzialmy :P )

Reply:
w sumie racja :)